src/Compilers/VisualBasic/Portable/Lowering/UseTwiceRewriter.vb

```diff
@@ -232,9 +235,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Debug.Assert(first.IsLValue AndAlso second.IsLValue)
             Return New Result(first, second)
-#Else
-            Return UseTwiceLValue(containingMember, node, arg)
-#End If
+
+        End Function
+
+        Private Shared Function IsInvariantArray(type As TypeSymbol) As Boolean
+            Dim value = TryCast(type, ArrayTypeSymbol)?.ElementType?.IsNotInheritable
```

Comment:
>? [](start = 67, length = 1)

The second conditional access feels redundant.

Reply:
Updated.